### DISCUSSION TITLE
%E4%B8%AD%E9%97%B4%E4%BB%B6/es/%E6%A0%B8%E5%BF%83%E7%9F%A5%E8%AF%86%E7%82%B9/elasticSearch%E6%9E%B6%E6%9E%84%E6%98%AF%E6%80%8E%E4%B9%88%E6%A0%B7%E7%9A%84

### DISCUSSION THREAD

**Discussion by giscus** [xiaobaiTech/golang-full-stack#20] [Announcements]
# %E4%B8%AD%E9%97%B4%E4%BB%B6/es/%E6%A0%B8%E5%BF%83%E7%9F%A5%E8%AF%86%E7%82%B9/elasticSearch%E6%9E%B6%E6%9E%84%E6%98%AF%E6%80%8E%E4%B9%88%E6%A0%B7%E7%9A%84

elasticSearch 是什么？工作原理是怎么样的？ 现在有三段文本，id 分别是 0、1、2，你需要快速找到哪段文本里含有关键词

https://golangguide.top/%E4%B8%AD%E9%97%B4%E4%BB%B6/es/%E6%A0%B8%E5%BF%83%E7%9F%A5%E8%AF%86%E7%82%B9/elasticSearch%E6%9E%B6%E6%9E%84%E6%98%AF%E6%80%8E%E4%B9%88%E6%A0%B7%E7%9A%84.html

**Comment by 0xiaobao0** [comment]
大佬，图片挂了，可以更新下吗~

**Reply by 0xiaobao0** [reply]
好像恢复了

**Reply by xiaobaiTech** [reply]
感谢提醒，是ssl过期了，刚修复

**Comment by PacketDragonStar** [comment]
大佬，图片又挂了😭
